Dockerfile

```diff
@@ -1,5 +1,5 @@
 ############# builder
```

Comment:
Do you also want to update to alpine:3.12.1 now that you are on it?

Reply:
Sure